stow/sass/_extra-child-theme.scss

```diff
@@ -202,10 +207,18 @@ a {
 	}
 }
 
+/**
+ * 4.2 Social Navigation
+ */
+.social-navigation > div > ul {
+	margin-left: 0;
```

Comment:
@jeffikus Looks like the white-space-remover plugin I used in Coda, made this diff bigger than it needed to be. All the magic happens with `0` margins on these two lines. :-)

Reply:
LGTM